contracts/lib/MultiTokenTransferLib.sol

```diff
@@ -0,0 +1,107 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.11;
+
+import "./CurrencyTransferLib.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC721/IERC721Upgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC1155/IERC1155Upgradeable.sol";
+
+library MultiTokenTransferLib {
+
+    struct Bundle {
+        address[] erc1155AssetContracts;
+        uint256[][] erc1155TokensToWrap;
+        uint256[][] erc1155AmountsToWrap;
+        address[] erc721AssetContracts;
+        uint256[][] erc721TokensToWrap;
+        address[] erc20AssetContracts;
+        uint256[] erc20AmountsToWrap;
+    }
+
+    function transferBundle(
+        address _from,
+        address _to,
+        Bundle memory _bundle
+    ) public {
+        transfer1155(_from, _to, _bundle);
+        transfer721(_from, _to, _bundle);
+        transfer20(_from, _to, _bundle);
+    }
+
+    function transfer20(
```

Comment:
i think we should opt for clearer name for functions.
`transferERC20`

Reply:
i wonder if it should be `transferERC20Bundle` since it's requiring bundle

Reply:
I think `transferERC20` is fine; the 'bundle' concept is clear from the function being a part of a library called 'MultTokenTransferLib', for which you'd want to check function signatures before using, anyway.

Reply:
sgtm